activerecord/lib/rails/generators/active_record/migration/migration_generator.rb

```diff
@@ -28,7 +28,7 @@ def create_migration_file
         def set_local_assigns!
           @migration_template = "migration.rb"
           case file_name
-          when /^(add|remove)_.*_(?:to|from)_(.*)/
+          when /^(add)_.*_(?:to)_(.*)/, /^(remove)_.*?_(?:to|from)_(.*)/
```

Comment:
The same change (and test) should be made to allow `migration = "remove_email_address_from_sent_to_user"`

The uncaptured groups can then be removed entirely:
```ruby
when /^(add)_.*_to_(.*)/, /^(remove)_.*_from_(.*)/
```